entrypoint.sh

```diff
@@ -32,8 +32,6 @@ then
     exit 0
 fi
   sleep 10
-  LOGGING_FILENAME=historical_server_stats_$SERVER_NUMBER.log python -m rcon.cli record_server_stats_inception &
```

Comment:
Maybe just comment this out? This stuff is getting re-enabled at some point I assume.

Reply:
it's fine to remove this. However why remove the code also?